### ISSUE TITLE
Add Support fro Monobook Skin

### ISSUE THREAD

**Issue by jpotter23** [Frederisk/Wikitext-Extension-Gadget#10]
The title says it all. The current scripts only work for Minerva users, would love to see this expanded to work with more skins. The ones that I favor are:

Monobook
Timeless
Vector
Chameleon

My current wiki uses Monobook so I would start the request there.

**Comment by Frederisk** [comment]
The skins that have been verified to be available include: vector, vector-2022, timeless, fandomdesktop, and minerva.
There is indeed a lack of verification on the compatibility of monobook and chameleon, but I don’t understand why you think:
> The current scripts only work for Minerva users.

**Comment by jpotter23** [comment]
A misconception on my part. I don't know Java all that well, and when I looked at the code, I only saw the isMinerva conditional and the bottom without any other conditions for specific skins. That and I can only see the Edit with VSCodium link when trying Minerva skin. I'll do some more testing with the skins you've mentioned. 

We currently use Monobook, so getting support for it would be my primary concern and request. However, I plan to eventually switch to Chameleon, so getting support on that skin would be great as well.

Regardless, I appreciate your time and effort on this wonderful project and apologize for the confusion on my part.
